src/content/docs/browser/new-relic-browser/configuration/experimental-features.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Experimental features in browser monitoring'
+metaDescription: "Opt-in to use experimental features in New Relic browser monitoring before they're generally available."
+freshnessValidatedDate: never
+---
+
+New Relic Browser offers a range of features to enhance your application monitoring capabilities. To ensure stability and reliability, some features are released in an experimental state. These features are subject to change, may not be fully supported, and should be used with caution.
```

Comment:
New Relic browser agent offers a range of features to enhance your application monitoring capabilities. Some features are released in experimental state to ensure stability and reliability. These features may change or may lack support. You must use them with caution.